ctam/test_runner.py

```diff
@@ -279,6 +285,7 @@ def _start(self, testrun_name="initialization"):
             logger=dut_logger,
             test_info_logger=test_info_logger,
             test_uri_response_check=self.test_uri_response_check,
```

Comment:
@v-sourabhsingh I see that CompToolDut requires a mandatory argument `workspace_dir`.  How were the changes tested? I don't see that it's been passed here and I am facing issue while running it. Thanks.

Reply:
It's been fixed. Please pull latest changes.